src/master-playlist-controller.js

```diff
@@ -371,6 +390,14 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.blacklistCurrentPlaylist(this.mainSegmentLoader_.error());
     });
 
+    this.mainSegmentLoader_.on('syncinfoupdate', () => {
+      this.onSyncInfoUpdate_();
+    });
+
+    this.audioSegmentLoader_.on('syncinfoupdate', () => {
+      this.onSyncInfoUpdate_();
+    });
```

Comment:
Sounds good. I think long run we have to look at combining the separate SyncControllers between segment loaders so that they can use the same information when choosing and placing segments. Specifically, I am thinking about the mapping of timestamps to display time.